lua/lazyvim/plugins/extras/coding/supermaven.lua

```diff
@@ -0,0 +1,20 @@
+return {
+  {
+    "supermaven-inc/supermaven-nvim",
+    config = function()
+      require("supermaven-nvim").setup({
+        keymaps = {
+          accept_suggestion = "<C-y>",
+          clear_suggestion = "<C-]>",
+          accept_word = "<C-j>",
+        },
+      })
+    end,
```

Comment:
I believe it would be better if this would be in `opts` instead of `config`, so that users can overwrite them in their personal configuration easier instead of having to overwrite the whole `config` function.